airflow/utils/cli_action_loggers.py

```diff
@@ -121,8 +123,17 @@ def default_action_log(sub_command, user, task_id, dag_id, execution_date, host_
                     }
                 ],
             )
-    except Exception as error:
-        logging.warning("Failed to log action with %s", error)
+    except (OperationalError, ProgrammingError) as e:
+        expected = [
+            '"log" does not exist',  # postgres
+            "no such table",  # sqlite
+            "log' doesn't exist",  # mysql
+            "Invalid object name 'log'",  # mssql
+        ]
+        if getattr(e, "args", None) and not any(x in e.args[0] for x in expected):
```

Comment:
I don’t think `e.args` can ever be missing (it’s guaranteed by Python)

Reply:
ok cool :)  i felt a bit silly writing that but i wasn't sure and wanted to err on side of safety

Reply:
do i even need `if e.args`?  i.e. will there always be an `e.args[0]`?

Reply:
No that’s not guaranteed. `args` is just whatever init args the exception receives, so

```pycon
>>> e = Exception()
>>> e.args
()
```

Reply:
but i could still iterate that

Reply:
so this should be fine
```
        error_is_ok = any(x in e.args[0] for x in expected)
        if not error_is_ok:
            logging.warning("Failed to log action %s", e)
```
yeah?

Reply:
oh, no, hah

Reply:
ok -- updated. i think we should be good to go here.